packages/core/src/application.ts

```diff
@@ -241,6 +242,43 @@ export class Application extends Context implements LifeCycleObserver {
     this.add(binding);
     return binding;
   }
+
+  /**
+   * Add a service to this application.
+   *
+   * @param cls - The service or provider class
+   *
+   * @example
+   * ```ts
+   * @bind({scope: BindingScope.SINGLETON})
+   * export class LogService {
+   *   log(msg: string) {
+   *     console.log(msg);
+   *   }
+   * }
+   *
+   * const uuidv4 = require('uuid/v4');
+   * export class UuidProvider implements Provider<string> {
+   *   value() {
+   *     return uuidv4();
+   *   }
+   * }
+   *
+   * app.service(LogService);
+   * app.service(UuidProvider, 'uuid');
```

Comment:
Please add examples showing how to receive `LogService` and `uuid` via dependency injection. Also show what is the effect of providing a custom service name - where is that name used?